docs/ops.html

```diff
@@ -3819,6 +3819,12 @@ <h3>Provisioning the KRaft controller quorum</h3>
 # ZooKeeper client configuration
 zookeeper.connect=localhost:2181
 
+# The inter broker listener in brokers to allow KRaft controller send RPCs to brokers
+inter.broker.listener.name=PLAINTEXT
+
+# Maps listener names to security protocols. Please add the inter broker listener protocol mapping
```

Comment:
I'm fine we don't add this and let users to add it if necessary, because the error log should be clear when broker start up if there's any wrong/missing config.